launch.sh

```diff
@@ -1,19 +1,11 @@
 #!/usr/bin/env bash
 
-# Read in config.env file; error if not found
-if [ ! -f config.env ]; then
-    echo "config.env not found, please run setup.sh"
-    exit 1
+# Read in .env file; error if not found
+if [ ! -f .env ]; then
+    echo ".env not found, running setup.sh"
+    bash setup.sh
 fi
-source config.env
-
-export NUM_GPUS=${NUM_GPUS}
-export MODEL_DIR="${MODEL_DIR}"/"${MODEL}-${NUM_GPUS}gpu"
-export GPUS=$(seq 0 $(( NUM_GPUS - 1 )) | paste -sd ',')
+source .env
 
 # On newer versions, docker-compose is docker compose
-if command -v docker-compose > /dev/null; then
-    docker compose up
-else
-    docker-compose up
-fi
+docker compose up -d --remove-orphans || docker-compose up -d --remove-orphans
```

Comment:
I was having issues with `orphan` containers, as well as the `command -v docker-compose > /dev/null; then` resulting in `docker compose` leading to `docker: 'compose' is not a docker command.`